models/bob.js

```diff
@@ -16,6 +16,11 @@ const BobModel = mongoose.model('Bob', bobSchema);
 
 
 // Bob functions
+
+/**
+  Create and save a new bob in the db
+  @param {Object[]} bobData
```

Comment:
Is there a nice way in this docs markup to specify what the object should contain? In the code you can just look down but assuming things are autogenerated for the docs it'd be nice to give an overview of the data